### ISSUE TITLE
What to do about lambda_GC?

### ISSUE THREAD

**Issue by privefl** [privefl/bigsnpr#40]
Currently, lambda_GC(X2) = lambda_GC(Z)^2.
That can be confusing.
What to do about it?

**Comment by earosenthal** [comment]
I'm no expert, but it is my understanding that the inflation factor is based on the chi-square distribution (I think that is what you mean by X2). It looks like getLambdaGC may be calculating the square root of the inflation factor. Can you verify this? I calculated the inflation factor in my data using two ways (z-score, p-value) and compared it to what getLambdaGC output:

```
# using the z-scores
> chisq <-obj.gwas$score^2
> median(chisq)/qchisq(0.5,1)
[1] 0.9829162
```

```
# using the p-values 
> chisq <- qchisq(1-obj.gwas$p.value,1)
> median(chisq)/qchisq(0.5,1)
[1] 0.9806252
```

>```
 bigsnpr:::getLambdaGC(obj.gwas)
[1] 0.9902582
> (bigsnpr:::getLambdaGC(obj.gwas))^2
[1] 0.9806112
```

**Comment by privefl** [comment]
Yes, currently `big_univLinReg()` uses t-scores and `big_univLogReg()` uses Z-scores, so you need to square it.

But this is also my understanding that it is usually reported for chi-squares (X2), therefore this question/issue.

**Comment by privefl** [comment]
Also, maybe think about exporting the function to get the lambda_GC, which for now can be run using `bigsnpr:::getLambdaGC()`.
